shell/components/__tests__/FixedBanner.test.ts

```diff
@@ -0,0 +1,137 @@
+import { shallowMount } from '@vue/test-utils';
+import FixedBanner from '../FixedBanner.vue';
+import { ExtendedVue, Vue } from 'vue/types/vue';
+import { DefaultProps } from 'vue/types/options';
+
+const SETTING_NO_CONSENT = { value: '{"loginError":{"message":"","showMessage":"false"},"bannerHeader":{"background":" #EEEFF4","color":" #141419","textAlignment":"center","fontWeight":null,"fontStyle":null,"fontSize":"14px","textDecoration":null,"text":"SOME HEADER TEXT"},"bannerFooter":{"background":" #EEEFF4","color":" #141419","textAlignment":"center","fontWeight":null,"fontStyle":null,"fontSize":"14px","textDecoration":null,"text":"SOME FOOTER TEXT"},"bannerConsent":{"background":" #EEEFF4","color":" #141419","textAlignment":"center","fontWeight":null,"fontStyle":null,"fontSize":"14px","textDecoration":null,"text":null,"button":null},"showHeader":"true","showFooter":"true","showConsent":"false"}' };
+const SETTING_WITH_CONSENT = { value: '{"loginError":{"message":"","showMessage":"false"},"bannerHeader":{"background":" #EEEFF4","color":" #141419","textAlignment":"center","fontWeight":null,"fontStyle":null,"fontSize":"14px","textDecoration":null,"text":"SOME HEADER TEXT"},"bannerFooter":{"background":" #EEEFF4","color":" #141419","textAlignment":"center","fontWeight":null,"fontStyle":null,"fontSize":"14px","textDecoration":null,"text":"SOME FOOTER TEXT"},"bannerConsent":{"background":" #EEEFF4","color":" #141419","textAlignment":"center","fontWeight":null,"fontStyle":null,"fontSize":"14px","textDecoration":null,"text":"SOME CONSENT TEXT" ,"button": "some-button-label"},"showHeader":"true","showFooter":"true","showConsent":"true"}' };
+
+const parsedBannerStyle = {
+  'background-color': ' #EEEFF4',
+  color:              ' #141419',
+  'font-size':        '14px',
+  'font-style':       '',
+  'font-weight':      '',
+  'text-align':       'center',
+  'text-decoration':  '',
+};
+
+const parsedBannerContent = {
+  background:     ' #EEEFF4',
+  color:          ' #141419',
+  fontSize:       '14px',
+  fontStyle:      null,
+  fontWeight:     null,
+  textAlignment:  'center',
+  textDecoration: null,
+};
+
+describe('component: FixedBanner', () => {
+  it('should render HEADER fixed banner correctly', async() => {
+    const wrapper = shallowMount(FixedBanner as unknown as ExtendedVue<Vue, {}, {}, {}, DefaultProps>, {
```

Comment:
I tried to remove `as unknown` instruction in my local environment, and it doesn't generate any linter errors, so probably we can remove it.

Reply:
If you try to build the application, then you get errors that you don't get using VSCode lint...

Reply:
> Hi Alex, good work here, I found some code that will be useful to me! 😃
> 
> I found something that maybe can be improved and I left some comments. Please let me know if you want to do any double checks.
> 
> My general considerations/preferences are the following, please feel free to share your thoughts about them:
> 
> * We should probably avoid as much as possible to put more than one `expect` in each test, for example in the case we want to test `emitters`. This will make code more readable and more easy to investigate in case of failings.
> * Use `data-test` instead of class selectors, specially in `v-for` elements .
> * Grouping tests using multiple `describe()` blocks improves readability and make it easier to change test code.

Hi @torchiaf . Thanks for reviewing the PR 🙇 

just a couple of comments on the points you mentioned:
> * We should probably avoid as much as possible to put more than one `expect` in each test, for example in the case we want to test `emitters`. This will make code more readable and more easy to investigate in case of failings.
I don't agree because I believe unit tests should cover more than a just a simple expect. Often code is spread out between different `methods` and `computed` props and I think it's relevant to test the chain of logic and see if it is respected, otherwise you might be passing the test and don't even know why OR you are not capturing relevant logic points that have a significant impact on how the component should work. 
Again, this is only my opinion and I believe that's why we should have guidelines for unit tests as well and discuss this as a team. Is it verbose? yes. Is it thoroughly tested? I believe so. Let me know if this is a blocker

> * Use `data-test` instead of class selectors, specially in `v-for` elements .
Not a fan, but it's an industry pattern. Let's use it.

> * Grouping tests using multiple `describe()` blocks improves readability and make it easier to change test code.
Agree. I am not used to using it, so it's something that I never do, but I do agree with you.